src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```
@@ -22,6 +22,7 @@ imports:
     - { resource: "@SyliusCoreBundle/Resources/config/app/parameters.yml" }
     - { resource: "@SyliusCoreBundle/Resources/config/app/sylius.yml" }
     - { resource: "@SyliusCoreBundle/Resources/config/app/state_machine.yml" }
+    - { resource: "@SyliusCoreBundle/Resources/config/app/workflows.yml" }
```

```suggestion
    - { resource: "@SyliusCoreBundle/Resources/config/app/workflows.yaml" }
```

Newer files already use `yaml`, so no reason to fallback to `yml`